cdi/pom.xml

```diff
@@ -167,7 +166,8 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
-                        <Fragment-Host>org.hibernate.validator</Fragment-Host>
+                        <Bundle-SymbolicName>${hibernate-validator-cdi.module-name}</Bundle-SymbolicName>
```

Comment:
@gunnarmorling you mentioned making the bundle names consistent with the Java 9 module names in another issue so that's what I did. I'm not sure if it can cause compatibility issues though?

Reply:
Why is the `Fragment-Host` header removed? Do you not want this to be a fragment any longer? On the name, did we actually define one for CDI before? If not, it should be fine, but if we did define it, let's wait with changing it at least until the next minor release (6.1) (same for the engine bundle name).

Reply:
Mmmh, yeah, the fragment host shouldn't have been removed. Don't know how it went away. I'll reintroduce it.

About the name, we did not define one explicitly but the plugin defined one based on the Maven coordinates. So we're definitely changing something if we do so. Thus my question about compatibility.

Reply:
Ah ah, it didn't go away, it's just there vvvvvv

Reply:
Ah yeah. That for superfluous changes :)